docs/site/src/pages/reference/python-client/read-data.mdx

```diff
@@ -121,7 +121,9 @@ situations where it's necessary to process large volumes of data. Sometimes
 these reads may be too large to fit in memory.
 
 Synnax supports server side iterators that allow us to process large queries in
-consistently sized chunks:
+consistently sized chunks. To configure reading in chunks, pass in `chunk_size` to the
```

Comment:
Fix this - it makes it seem like you need to specify the chunk size in order to read in chunks. By default, Synnax uses a chunk size of X. If you'd like to set a custom size, use the `chunk_size` argument.